django/db/backends/postgresql/introspection.py

```diff
@@ -145,7 +142,6 @@ def get_indexes(self, cursor, table_name):
                 indexes[row[0]]['primary_key'] = True
             if row[2]:
                 indexes[row[0]]['unique'] = True
-            indexes[row[0]]['type'] = row[4]
         return indexes
 
     def get_constraints(self, cursor, table_name):
```

Comment:
Shouldn't the docstring be amended to add index_type? Now it says, "Retrieves any constraints or keys (unique, pk, fk, check, index) ..."